app/src/main/java/org/sopt/pingle/presentation/ui/plan/planopenchatting/PlanOpenChattingFragment.kt

```diff
@@ -16,15 +20,21 @@ class PlanOpenChattingFragment :
     private val planViewModel: PlanViewModel by activityViewModels()
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-
         binding.viewModel = planViewModel
 
         addListeners()
+        collectData()
     }
 
     private fun addListeners() {
         binding.root.setOnClickListener {
             requireActivity().hideKeyboard(it)
         }
     }
+
+    private fun collectData() {
+        planViewModel.planOpenChattingLink.flowWithLifecycle(lifecycle).onEach { openChattingLink ->
+            planViewModel.validityOpenChattingLink()
+        }.launchIn(lifecycleScope)
```

Comment:
```suggestion
        planViewModel.planOpenChattingLink.flowWithLifecycle(viewLifecycleOwner.lifecycle).onEach { openChattingLink ->
            planViewModel.validityOpenChattingLink()
        }.launchIn(viewLifecycleOwner.lifecycleScope)
```

프래그먼트에서 라이프 사이클을 주입할 때 한 번 더 생각합니둥 ㅋ.ㅋ 갠적으로 이거 참 중요한 개념이라 생각합니당